core/test/Size.js

```diff
@@ -45,7 +45,15 @@ test('Size', function(t) {
     });
 
     t.test('fromSpecWithParent method', function(t) {
-        // TODO
+        // TODO: Size.DEFAULT
+        
+        // TODO: Size.ABSOLUTE
+        
+        // TODO: Size.RELATIVE
+        
+        // TODO: Size.RENDER
+        
+        // TODO: Size.CUSTOM
```

Comment:
If someone could share a general framework / thought process for testing the different modes, I can take a stab at writing the tests.  We definitely need these as there was already one issue #354 & #355